wallet-new/src/Cardano/Wallet/API/V1/Wallets.hs

```diff
@@ -37,4 +37,16 @@ type API = Tags '["Wallets"] :>
                    :> Summary "Update the Wallet identified by the given walletId."
                    :> ReqBody '[ValidJSON] (Update Wallet)
                    :> Put '[ValidJSON] (WalletResponse Wallet)
+    :<|> "external-wallets"
+                   :> CaptureRootPK
+                   :> Summary "Check if this external wallet is presented in the node."
+                   :> PostCreated '[ValidJSON] (WalletResponse WalletAndTxHistory)
+    :<|> "external-wallets"
+                   :> Summary "Creates a new or restores an existing external wallet (mobile client or hardware wallet)."
+                   :> ReqBody '[ValidJSON] (New ExternalWallet)
+                   :> PostCreated '[ValidJSON] (WalletResponse Wallet)
+    :<|> "external-wallets"
+                   :> CaptureRootPK
+                   :> Summary "Deletes the given external wallet and all its accounts."
+                   :> DeleteNoContent '[ValidJSON] NoContent
```

Comment:
I'd avoid type aliases here `CaptureRootPK` especially if it's in another file :/ ... It increases the cognitive load for not-so-much added value in the end :/